app/models/team.rb

```diff
@@ -564,8 +564,10 @@ def filter_by_keywords(query, filters, type = 'FactCheck')
   end
 
   def search_for_similar_articles(query, pm = nil)
-    # query expected to be text
-    result_ids = Bot::Smooch.search_for_similar_published_fact_checks_no_cache('text', query, [self.id]).map(&:id)
+    # query:  expected to be text
+    # pm: to request a most relevant to specific item and also include both FactCheck & Explainer
+    limit = pm.nil? ? CheckConfig.get(:most_relevant_team_limit, 3, :integer) : CheckConfig.get(:most_relevant_item_limit, 10, :integer)
```

Comment:
Do we have any case currently where this method is going to be called with a `nil` `pm`? I don't think so... and in that case, we can simplify this method and only handle the case where there is a `pm`, so we just need a single configuration key and we always return both fact-checks and explainers.

Reply:
There are two cases for calling this method:

1. From `TeamType`  the one should called by Jay 
2. From `ProjectMediaType`  to get most relevant 

case 1: should return FactCheck and fallback to Explainer if no FactCheck with limit 3
case 2: should return both FactCheck & Explainer  with limit 10

Reply:
Ah, you're right, my bad!